python/engine/test/PythonEngine_GTest.cpp

```diff
@@ -134,3 +135,38 @@ RecursionError: maximum recursion depth exceeded)",
     EXPECT_EQ(expected_exception, error);
   }
 }
+
+TEST_F(PythonEngineFixture, AlfalfaMeasure) {
+  const std::string classAndDirName = "AlfalfaMeasure";
+
+  const auto scriptPath = getScriptPath(classAndDirName);
+  auto measureScriptObject = (*thisEngine)->loadMeasure(scriptPath, classAndDirName);
+  auto* measurePtr = (*thisEngine)->getAs<openstudio::measure::ModelMeasure*>(measureScriptObject);
+
+  ASSERT_EQ(measurePtr->name(), "Alfalfa Measure");
+
+  std::string workflow_json = "\
+  {\
+    \"seed\": \"../seed.osm\",\
+    \"weather_file\": \"../weather.epw\",\
+    \"steps\": [\
+      {\
+        \"arguments\": {},\
+        \"description\": \"The method attempts to build an alfalfa json in the measure\",\
+        \"measure_dir_name\": \"AlfalfaMeasure\",\
+        \"modeler_description\": \"The method attempts to build an alfalfa json in the measure\",\
+        \"name\": \"AlfalfaMeasure\"\
+      }\
+    ]\
+  }";
```

Comment:
Use a Raw string litteral here. I'm also seeing extra empty lines, so you should use clang-format. (**Why did our CI not catch this?** the python/ folder is outside... geez)


```suggestion
  std::string workflow_json = R"json(
  {
    "seed": "../seed.osm",
    "weather_file": "../weather.epw",
    "steps": [
      {
        "arguments": {},
        "description": "The method attempts to build an alfalfa json in the measure",
        "measure_dir_name": "AlfalfaMeasure",
        "modeler_description": "The method attempts to build an alfalfa json in the measure",
        "name": "AlfalfaMeasure"
      }
    ]
  })json";
```